lib/src/main/java/com/hootsuite/nachos/NachoTextView.java

```diff
@@ -148,6 +149,7 @@ public void onChipClick(Chip chip, MotionEvent motionEvent) {
     // Touch events
     @Nullable
     private OnChipClickListener mOnChipClickListener;
+    private GestureDetector gestureDetector;
```

Comment:
Any reason it's a global variable? Since it's only listening for single taps, you could rename it to make it more descriptive, for example singleTapDetector.